PIPs/pip-27.md

```diff
@@ -117,6 +117,338 @@ To export data to a centralized server, we need to follow these procedures:
 3. The server will keep only the last 3 databases available for download and delete the rest.
 4. It will update a JSON object to show the available databases for download.
 
+### Backup script 
+
+This script first stops the Pactus node if it is running, then creates a backup of
+the blockchain data by copying or compressing it based on the specified options.
+The backup is stored in a timestamped snapshot directory along with a `metadata.json`
+file that contains detailed information about the snapshot, including file paths and checksums.
+Finally, the script manages the retention of snapshots, ensuring only a specified number of
+recent backups are kept.
+
+```python
+import argparse
```

Comment:
Should we keep the whole implementation on a proposal? I think it's better to add an interface to the proposal instead of an implementation. things can change in the software and protocol and we need to update the PIP always...

at most, we can put a reference link to this implementation instead of the code in the PIP. first time I can see this thing.

Reply:
@b00f @kehiy I think we put this script in `github.com/pactus-project/pactus/scripts/backup_tool.py` with README.md

Reply:
I agree, later we can put a reference on the PIP! good idea.